backend/ee/danswer/server/analytics/api.py

```diff
@@ -115,3 +118,78 @@ def get_danswerbot_analytics(
     ]
 
     return resolution_results
+
+
+class PersonaMessageAnalyticsResponse(BaseModel):
+    total_messages: int
+    date: datetime.date
+    persona_id: int
+
+
+@router.get("/admin/persona/messages")
+def get_persona_messages(
+    persona_ids: str = Query(...),  # ... means this parameter is required
+    start: datetime.datetime | None = None,
+    end: datetime.datetime | None = None,
+    _: User | None = Depends(current_admin_user),
+    db_session: Session = Depends(get_session),
+) -> list[PersonaMessageAnalyticsResponse]:
+    """Fetch daily message counts for multiple personas within the given time range."""
+    # Convert comma-separated string to list of integers
+    parsed_persona_ids = [
+        int(id.strip()) for id in persona_ids.split(",") if id.strip()
+    ]
+    persona_message_counts = []
+    start = start or (datetime.datetime.utcnow() - datetime.timedelta(days=30))
```

Comment:
For any magic numbers would prefer to declar a constant somewhere that we reference